src/style.css

```diff
@@ -1 +1,118 @@
-/* add styles here */
+/* #region paje */
+
+@font-face {
+  font-family: avenir;
+  src: url(fonts/Avenir-Book.ttf), url(fonts/Avenir.ttc);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+* {
```

Comment:
Using the universal selector '*' is not recommended as it impacts performance. Instead, apply styles to specific elements or classes that require them.